packages/edit-site/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Enhancements
+
+-   Add ´@wordpress/html-entities´ package to the list of dependencies in package.json. ([#62313](https://github.com/WordPress/gutenberg/pull/62313))
```

Comment:
This PR is filed as a bug fix, but here it's filed an enhancement.

Reply:
Thank you, what a lazy mistake from my side!